config/pokeapi.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"fmt"
+	"go-bootcamp/usecase/clients"
```

Comment:
Try to keep, the following order on your imports,

Core packages

Your packages

Third party packages